src/config/dynamic-theme-fixes.config

```diff
@@ -18596,6 +18650,14 @@ www.mayoclinic.org
 INVERT
 .mc-logo
 .logo > a > img[alt="Mayo Clinic"]
+img[src="/-/media/images/mayologo.png"]
+img[src="/styles/img/gbs/logo-mayoclinic-mobile.png"]
+
+CSS
+ul#nav.nav > li.no-image > a,
+ul#nav.nav > li.current > a {
```

Comment:
Could `IGNORE IMAGE ANALYSIS` be used here?

Reply:
Doesn't seem to work, the icons are still replaced by other icons:
![imagen](https://user-images.githubusercontent.com/71190696/163284675-eccec961-9294-49a5-9cd6-14f23c4639f4.png)

If it should work, I may be doing something wrong then.

Reply:
Try to refresh the site a few times(it can take a refresh to have actual effect)

Reply:
Still doesn't work for me after reloading it a bunch of times on both Chrome and Firefox, I even deleted site data and everything.

Just in case, I'm testing on www.mayoclinic.org/symptoms with
```
IGNORE IMAGE ANALYSIS
ul#nav.nav > li.no-image > a
ul#nav.nav > li.current > a
```